src/inc/regdisp.h

```diff
@@ -11,7 +11,7 @@ class Thread;
 #endif // DEBUG_REGDISPLAY
 
 
-#if defined(_TARGET_X86_)
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
```

Comment:
Why is the REGDISPLAY commented out for WIN64EXCEPTIONS? It is a structure that is generally needed.